src/common/styles/antd-overrides.scss

```diff
@@ -35,6 +35,10 @@
 
       .ant-table-tbody {
         .ant-table-cell {
+          a {
+            max-width: 420px;
```

Comment:
@miteshsawant Have added this because links were long and the table was scrolling the page